package.json

```diff
@@ -48,6 +48,8 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-release": "^3.2.0",
     "esprima-fb": "^8001.2001.0-dev-harmony-fb",
+    "husky": "^8.0.3",
```

Comment:
Can we use `yorkie`, like in the eslint/eslint project? The setup looks much simpler, and when we were introducing pre-commit hooks to eslint/eslint, we opted for `yorkie` (ref discussion on https://github.com/eslint/eslint/pull/11556#discussion_r269373909).